requirements/uninstall-requirements.txt

```diff
@@ -10,3 +10,4 @@ zeep
 cython
 django-celery-results
 numpy
+ConcurrentLogHandler
```

Comment:
There's a weird thing for these type of libraries that were taken over by someone else. Its possible that they use the same folder in site-packages, so uninstalling this might actually uninstall concurrent-log-handler. Can you confirm that if you install `concurrent-log-handler` and then uninstall `ConcurrentLogHandler` still works?

Reply:
Yep I checked that.. They use different namespaces thankfully so it wasn't an issue (`cloghandler` vs `concurrent_log_handler`).

Reply:
er, i don't know if the namespace thing is the reason, but it did in fact work when i did it in that order

Reply:
awesome. thanks for verifying that